LWCE_Core/Src/XComLongWarCommunityEdition/Classes/LWCE_UITacticalHUD_Radar.uc

```diff
@@ -29,6 +67,10 @@ simulated function UpdateActiveUnit()
     {
         Show();
     }
+    else if ((kActiveUnit.m_iZombieMoraleLoss & -2147483648) != 0)
```

Comment:
Shoot, I thought this was added by the Sightlines mod. I accidentally had it installed when I decompiled the LW source and it's been haunting me ever since. No wonder this broke. Good catch.